.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# Dependabot configuration.
+#
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
```

Comment:
```suggestion
```

The [default value](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit) is 5 already.

Reply:
True, except most people don't know that and the default may change without notice.
This makes it explicit and easy to check for people. Happy to remove it if you prefer, but at least you now know my reasoning for including it.

Reply:
We already rely on other defaults (like it being on a Monday), so this is no different. Having it also suggests that we're limiting it on purpose (I was going to query why we'd want to do that, really), rather than leaving it to the defaults (more effort to look up).

Reply:
So, what should we do ?

Reply:
I'm really fine either way. Extra information doesn't hurt  I guess